### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by annstriganova**
I've been tried to reduce duplicate constructions for taking screenshot on error. 
Now, Instead of this in each test class
```java
try {
    //Assertions
} catch (Throwable err) {
    String filepath = AtsdTest.screenshotDir + "/" + this.getClass().getSimpleName() + "_"
    + Thread.currentThread().getStackTrace()[1].getMethodName() + "_" + System.currentTimeMillis()
    + ".png";
    this.saveScreenshot(filepath);
    throw err;
}
``` 
there is an [ActionOnTestState](https://github.com/axibase/atsd-web-test/blob/00d69492ba3e823be901e1c163974d4fa6936d1e/atsd_webtests/src/test/java/com/axibase/webtest/service/ActionOnTestState.java#L20) rule in base class [AtsdTest](https://github.com/axibase/atsd-web-test/blob/00d69492ba3e823be901e1c163974d4fa6936d1e/atsd_webtests/src/test/java/com/axibase/webtest/service/AtsdTest.java#L25). 
I'm not sure that it's much better than previous solution with try-catch, because there is an important restriction - it's impossible to perform any actions with driver in `@After` methods.

### CODE REVIEW

**Comment by raipc** [comment]
I have never seen the screenshots from web tests. Do we really need them?

**Comment by annstriganova** [comment]
@raipc, yes, they are useful, for example, when WebElement is not found.

File: atsd_webtests/atsd.properties [hunk 1]
```diff
@@ -2,4 +2,4 @@ url=http://localhost:8088
 login=axibase
 password=axibase
 screenshot_directory=/tmp/atsd_webtests
-phantomjs.binary.path=/opt/bin/phantomjs
```
Comment (VeselovAlex) [inline atsd_webtests/atsd.properties]: Please revert this file, it was not actually changed

File: atsd_webtests/src/test/java/com/axibase/webtest/cases/CreateEntityAndMetricTest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        driver.findElement(By.name("value")).sendKeys("150");
+        driver.findElement(By.name("series")).click();
+        successMessage = driver.findElement(By.cssSelector("form.series .form-status>.alert-success"));
+        assertEquals(generateAssertMessage("Wrong success message"), "Series inserted successfully", successMessage.getText().trim());
+
+        driver.navigate().to(AtsdTest.url + "/portals/series?entity=my-entity&metric=my-metric");
+        List<WebElement> widgets = driver.findElements(By.className("widget"));
+        assertNotEquals(generateAssertMessage("No widgets for portal"), 0, widgets.size());
+    }
+}
```
Comment (VeselovAlex) [inline atsd_webtests/src/test/java/com/axibase/webtest/cases/CreateEntityAndMetricTest.java]: Add new line for consistency

File: atsd_webtests/src/test/java/com/axibase/webtest/cases/CreateUserGroupTest.java [hunk 3]
```diff
@@ -2,13 +2,14 @@
...
+import org.junit.*;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
 /**
  * Created by Anna Striganova on 11.04.18.
```
Comment (VeselovAlex) [inline atsd_webtests/src/test/java/com/axibase/webtest/cases/CreateUserGroupTest.java]: Remove this comment please, Git always knows who is an author and when it was created :)

**Pull request merged**